Level_1/pick_game.py

```diff
@@ -0,0 +1,27 @@
+def solution(board, moves):
+    answer = 0
+    arr = []
+    bucket = []
+
+    # column list 생성
+    for i in range(len(board[0])):
+        line = []
+        for j in range(len(board)):
+            if board[j][i] != 0:
+                line.append(board[j][i])
```

Comment:
여기는 한줄로 더 보기좋게 되네용
```suggestion
        line = [board[j][i] for j in range(len(board))  if board[j][i] != 0]           
```
이렇게 하면 사실 arr 까지 2중 리스트 컴프리헨션도 가능하죠 ㅎㅎ
```python
    arr = [
        [board[j][i] for j in range(len(board)) if board[j][i] != 0] 
        for i in range(len(board))
    ]
```

Reply:
리스트 컴프리헨션도 자주 써봐야 겠네요!!